doc/manual/src/contributing/hacking.md

```diff
@@ -35,6 +35,25 @@ variables are set up so that those dependencies can be found:
 $ nix-shell
 ```
 
+or if you have a flake-enabled nix:
+
+```console
+$ nix develop
+```
+
+To get a shell with a different compilation environment (e.g. stdenv,
+gccStdenv, clangStdenv, clang11Stdenv):
+
+```console
+$ nix-shell -A devShell.x86_64-linux.clang11StdenvPackages
+```
+
+or if you have a flake-enabled nix:
+
+```console
+$ nix develop .#devShell.x86_64-linux.clang11StdenvPackages
```

Comment:
This should be `.#clang11StdenvPackages` now right?

Reply:
Indeed, thanks. And actually the flake search path was broken (was missing a trailing `.`), so I fixed it.

I’ve also fixed the command above to use `devShells` and not `devShell`